maps/random_ruins/exoplanets/adhomai/adhomai_village.dmm

```diff
@@ -436,8 +457,16 @@
 /obj/structure/table/rack,
 /obj/item/towel,
 /obj/item/soap/uranium_soap,
-/turf/simulated/floor/tiled/freezer{
-	temperature = 268.15
+/obj/effect/floor_decal/corner/blue/diagonal,
+/turf/simulated/floor/tiled/white{
+	temperature = 278.15
+	},
```

Comment:
Why? If it's a cold room, there's the tiles for it, as it was before

Reply:
floor is changed back to how it was now